src/wheel/bdist_wheel.py

```diff
@@ -69,8 +68,8 @@ def get_flag(var, fallback, expected=True, warn=True):
 
 
 def get_abi_tag():
-    """Return the ABI tag based on SOABI (if available) or emulate SOABI (PyPy)."""
-    soabi = get_config_var("SOABI")
+    """Return the ABI tag based on SOABI (if available) or emulate SOABI (PyPy2)."""
```

Comment:
(not really related, just noticed this) Is PyPy2 really still supported by wheel? I thought it requires Python 3.7+?

Reply:
I think all platforms these days have a valid SOABI, even windows. So "PyPy2" is more of a generic placeholder for "weird platform".